### DISCUSSION TITLE
CIS cluster scan fails to execute

### DISCUSSION THREAD

**Discussion by jacobcrawford** [aquasecurity/trivy#5933] [Bugs]
### Description

I try to run the following command: `trivy k8s cluster --compliance=k8s-cis --report summary` on an AKS cluster v1.26.6.
And the error thrown does not really give me any helping context as to how to fix the problem:
![image](https://github.com/aquasecurity/trivy/assets/22328547/33ea5ed0-01fc-493a-a570-d306aba864c4)

### Desired Behavior

Trivy runs a cis-benchmark on the cluster and outputs the result to the terminal.
If an error exists on my side I expect an informative error message.

### Actual Behavior

Trivy CLI throws an error that i do not understand

### Reproduction Steps

```bash
1. trivy k8s cluster --compliance=k8s-cis --report summary
```

### Target

Kubernetes

### Scanner

None

### Output Format

Table

### Mode

None

### Debug Output

```bash
2024-01-12T13:17:42.419+0100    FATAL   get k8s artifacts with node info error:
    github.com/aquasecurity/trivy/pkg/k8s/commands.clusterRun
        /home/runner/work/trivy/trivy/pkg/k8s/commands/cluster.go:38
  - could not parse reference: ToBeReplaced:0.0.1
```

### Operating System

Windows WSL

### Version

```bash
Version: 0.48.2
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-01-12 06:12:18.90121573 +0000 UTC
  NextUpdate: 2024-01-12 12:12:18.901215459 +0000 UTC
  DownloadedAt: 2024-01-12 09:54:00.405072001 +0000 UTC
Policy Bundle:
  Digest: sha256:8bfc31f3e4301ef758b6793a07e0b12b4306e0b54d03a640efb2ff5e5ef29b9e
  DownloadedAt: 2024-01-12 09:54:01.329786576 +0000 UTC
```

### Checklist

- [X] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by chen-keinan** [comment]
@jacobcrawford converting this discussion to #5943 for investigation
